leeanp.php

```diff
@@ -0,0 +1,27 @@
+<?php namespace LeanWoocommerceApi;
+/**
+ * Plugin Name: Lean Woocommerce Api
+ * Description: Woocommerce API endpoints for Lean.
+ * Version: 0.1.0
+ * Author: Moxie
+ * Author URI: http://getmoxied.net
+ * Text Domain: leanwoocommerceapi
+ */
+
+// General constants.
+define( 'LEANWOOCOMMERCEAPI_PLUGIN_NAME', 'LeanWoocommerceApi Plugin' );
```

Comment:
I think constants name should be separate by underscores like: `LEAN_WOOCOMMERCE_API_PLUGIN_NAME`